tests/kafkatest/services/kafka/kafka.py

```diff
@@ -874,7 +874,7 @@ def start_node(self, node, timeout_sec=60, **kwargs):
         self.logger.info("kafka.properties:")
         self.logger.info(prop_file)
         node.account.create_file(KafkaService.CONFIG_FILE, prop_file)
-        node.account.create_file(self.LOG4J_CONFIG, self.render('log4j.properties', log_dir=KafkaService.OPERATIONAL_LOG_DIR))
+        node.account.create_file(self.LOG4J_CONFIG, self.render('log4j2.properties', log_dir=KafkaService.OPERATIONAL_LOG_DIR))
```

Comment:
ditto

Reply:
Please noted that the previous version should use `-Dlog4j.configuration` and trunk version should use `-Dlog4j2.configurationFile`

Reply:
It seems that tons of code needs to apply this change.
I will prepare it asap.